series/series.go

```diff
@@ -818,3 +818,23 @@ func (s Series) Sum() float64 {
 	}
 	return sum
 }
+
+// Slice slices Series from j to k-1 index.
+func (s Series) Slice(j, k int) Series {
+	if err := s.Err; err != nil {
+		return s
+	}
+
+	if j > k {
+		empty := s.Empty()
+		empty.Err = fmt.Errorf("slice index out of bounds")
+		return empty
+	}
+
+	indixes := []int{}
```

Comment:
This can be slow for big slices. It's better to use `make([]int, k-j)` and assign later `indices[..] = ..`.